content/ja/docs/setup/production-environment/tools/kops.md

```diff
@@ -56,10 +56,10 @@ To download a specific version, replace the following portion of the command wit
 $(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)
 ```
 
-For example, to download kops version v1.15.0 type:
+For example, to download kops version v1.20.0 type:
 
 ```shell
-curl -LO  https://github.com/kubernetes/kops/releases/download/1.15.0/kops-darwin-amd64
+curl -LO https://github.com/kubernetes/kops/releases/download/1.20.0/kops-darwin-amd64
```

Comment:
[In original PR](https://github.com/kubernetes/website/pull/27570/files#diff-93e2efd4dc6f52998817f78e73f359bc5c3eaa2aac381b99f281b88a7d5de2edR62), `v` prefix is included to the version tag.
We should updates here with `v1.20.0` similarly?

Reply:
done